[AutoPR Az.cosmos-db.DefaultTag] Update dataTransferService.json

azure-sdk opened the pull request:
Create to sync https://github.com/Azure/azure-rest-api-specs/pull/31960


 This pull request has been automatically generated for preview purposes.

jorgeferreir2k21 commented:
> Create to sync https://github.com/Azure/azure-rest-api-specs/pull/31960
> 
> 
>  This pull request has been automatically generated for preview purposes.

Commit:
CodeGen from PR 31960 in Azure/azure-rest-api-specs
Merge c248cb46577cac5cdf0ee14f4cc136ca851f9de6 into b228f178ed852d198497e797789da6ac05af09c0
